Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM debian:jessie
-MAINTAINER Benjamin Böhmke
+MAINTAINER Benjamin Boehmke <benjamin@boehmke.net>; Dennis Twardowsky <twardowsky@gmail.com>
```

Comment:
Maybe remove this completely because it is [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)?

Reply:
Will be removed with the next commit.